test/proxy/ERC1967/ERC1967Utils.test.js

```diff
@@ -21,152 +21,152 @@ contract('ERC1967Utils', function (accounts) {
-    });
-
-    describe('getBeacon', function () {
-      it('returns current beacon and matches beacon slot value', async function () {
-        expect(await this.utils.$getBeacon()).to.equal(this.beacon.address);
-        expect(await getAddressInSlot(this.utils.address, BeaconSlot)).to.equal(this.beacon.address);
-      });
-    });
-
-    describe('upgradeBeaconToAndCall', function () {
-      it('sets beacon in storage and emits event', async function () {
-        const newBeacon = await UpgradeableBeaconMock.new(this.v2.address);
-        const receipt = await this.utils.$upgradeBeaconToAndCall(newBeacon.address, EMPTY_DATA);
-
-        expect(await getAddressInSlot(this.utils.address, BeaconSlot)).to.equal(newBeacon.address);
-        expectEvent(receipt, 'BeaconUpgraded', { beacon: newBeacon.address });
-      });
-
-      it('reverts when beacon does not contain code', async function () {
-        await expectRevertCustomError(
-          this.utils.$upgradeBeaconToAndCall(anotherAccount, EMPTY_DATA),
-          'ERC1967InvalidBeacon',
-          [anotherAccount],
-        );
-      });
-
-      it("reverts when beacon's implementation does not contain code", async function () {
-        const newBeacon = await UpgradeableBeaconMock.new(anotherAccount);
-
-        await expectRevertCustomError(
-          this.utils.$upgradeBeaconToAndCall(newBeacon.address, EMPTY_DATA),
-          'ERC1967InvalidImplementation',
-          [anotherAccount],
-        );
-      });
-
-      describe('when data is empty', function () {
-        it('reverts when value is sent', async function () {
-          const newBeacon = await UpgradeableBeaconMock.new(this.v2.address);
-          await expectRevertCustomError(
-            this.utils.$upgradeBeaconToAndCall(newBeacon.address, EMPTY_DATA, { value: 1 }),
-            'ERC1967NonPayable',
-            [],
-          );
-        });
-      });
-
-      describe('when data is not empty', function () {
-        it('delegates a call to the new implementation', async function () {
-          const initializeData = this.v2.contract.methods.mockFunction().encodeABI();
-          const newBeacon = await UpgradeableBeaconMock.new(this.v2.address);
-          const receipt = await this.utils.$upgradeBeaconToAndCall(newBeacon.address, initializeData);
-          await expectEvent.inTransaction(receipt.tx, await V2.at(this.utils.address), 'MockFunctionCalled');
-        });
-      });
-
-      describe('reentrant beacon implementation() call', function () {
-        it('sees the new beacon implementation', async function () {
-          const newBeacon = await UpgradeableBeaconReentrantMock.new();
-          await expectRevertCustomError(
-            this.utils.$upgradeBeaconToAndCall(newBeacon.address, '0x'),
-            'BeaconProxyBeaconSlotAddress',
-            [newBeacon.address],
-          );
-        });
-      });
-    });
-  });
+  // describe('IMPLEMENTATION_SLOT', function () {
```

Comment:
Are all these lines to be removed or uncommented?

Reply:
Those are to be removed.